plugins/modules/ssm_parameter.py

```diff
@@ -11,7 +11,7 @@
 short_description: Manage key-value pairs in AWS Systems Manager Parameter Store
 description:
   - Manage key-value pairs in AWS Systems Manager (SSM) Parameter Store.
-  - Prior to release 5.0.0 this module was called C(community.aws.aws_ssm_parameter_store).
+  - Prior to release 5.0.0 this module was called C(amazon.aws.aws_ssm_parameter_store).
```

Comment:
This should not be part of this PR. It will be done with the promotion of `amazon.aws.ssm_paramater` module